docs/esa/jwst/jwst.rst

```diff
@@ -524,11 +481,11 @@ A table can be uploaded to the server in order to be used in a query.
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> upload_resource='mytable.xml.gz'
-  >>> j=Jwst.launch_job(query="SELECT * from tap_upload.table_test", \
-  >>>   upload_resource=upload_resource, \
-  >>>   upload_table_name="table_test", verbose=True)
-  >>> r=j.get_results()
+  >>> upload_resource = 'mytable.xml.gz'
+  >>> j = Jwst.launch_job(query="SELECT * from tap_upload.table_test",
+  >>>                   upload_resource=upload_resource,
```

Comment:
same here, use `...` instead of `>>>` for continued lines. 

I would say this is the only issue I need to have fixed for merging

Reply:
Fixed